### ISSUE TITLE
Empty comment isn't focused when annotation is clicked

### ISSUE THREAD

**Issue by mrtcode** [zotero/zotero#4779]
https://forums.zotero.org/discussion/119025/cursor-focus-annotation-note-text

@abaevbog Could you remind me why only `note` annotation is allowed [here](https://github.com/zotero/reader/blob/ffc5593584aa858f5a4de78197a75066a19de504/src/common/reader.js#L1198)? Is focusing on other annotation type comments somehow interfering with VPAT stuff?

**Comment by abaevbog** [comment]
The main overarching idea from the a11y viewpoint is that focus movement not initiated by the user is generally bad, as it causes disorientation. And if the focus does get moved, it should be consistent and predictable. It is better to avoid the situation where if you select an annotation, _sometimes_ you'll be able to move/resize the annotation and _some other times_, you'll be taken to the comment field. 

While trying this out with a screen reader, I realized that notes might be a bit of an exception because the note by itself doesn't actually mean anything and it's comment is the only thing that matters. So if one wants to add a note, they will press the shortcut and expect to type the note right away. Otherwise, they will have just an empty meaningless note annotation and to edit it (which is the only thing you want to do with an empty note) one would have to leave that annotation they just made, go all the way to the sidebar, find and edit it there. So it would be an unnecessary multi-step process. (For that scenario, though, hiding the sidebar helped because that way you can get to the comment field in the popup, as opposed to going to the sidebar, but this is something that would have to be discovered.) Treating the note in a special way is intended to avoid those unnecessary steps and to provide a similar interface to text annotations, where you press a shortcut and you can immediately type.

I believe the intention was to only do that when a note annotation is added (or I suppose selected too, since why else would one select an empty note) via keyboard. I don't think there is any strong reason to not focus the annotation comment after mouse click, which seems to be the regression we have now.

**Comment by mrtcode** [comment]
It should now focus on a comment only when clicked. This applies to all annotations. Please check if it works as expected on your end.

**Comment by abaevbog** [comment]
@mrtcode now, the comment field gets focused  when the annotation is selected via click but not via Enter keypress, which is what we are after.

However, if a new note annotation is now created by pressing a shortcut Control-Option-3, the comment for the note no longer gets focused. It was part of the idea described above to treat the notes as a special case and focus the comment when they are created via shortcuts, so that one does not have to navigate to the sidebar to add text to an empty note they just added.

**Comment by mrtcode** [comment]
Ok, I'll will fix that.
